README.md

```diff
@@ -156,6 +156,18 @@ Below is the expected structure of such a config file:
   }
 }
 ```
+NOTE: Make sure to mint enough(threshold is 80,000 MKR) tokens for launch if you are provide it in `imports.gov`:
+```
+    sethSend "$MCD_GOV" 'mint(address,uint256)' "$ETH_FROM" "$(seth --to-uint256 "$(seth --to-wei 1000000 ETH)")"
+```
+NOTE: Make sure to `launch` MCD_ADMIN if you are provide it in `imports.authority`.
+```
+# lock more then threshold (80,000 MKR)
```

Comment:
Why more? exact threshold value is enough.
https://github.com/makerdao/dss-gov/blob/master/src/DssGov.sol#L453